packages/elements/src/overlay-menu/index.ts

```diff
@@ -378,6 +384,10 @@ export class OverlayMenu extends Overlay {
   protected update (changedProperties: PropertyValues): void {
```

Comment:
change the function to `willUpdate` and move `super.willUpdate` to the top of the function

Reply:
fixed